src/Raven.Server/Utils/ChangeVectorUtils.cs

```diff
@@ -288,7 +288,11 @@ public static string MergeVectorsDown(List<string> changeVectors)
                 ChangeVectorParser.MergeChangeVectorDown(changeVectors[i], _mergeVectorBuffer);
             }
 
-            return _mergeVectorBuffer.SerializeVector();
+            var mergedChangeVector = _mergeVectorBuffer.SerializeVector();
+            if (string.Equals(changeVectors[0], mergedChangeVector, StringComparison.OrdinalIgnoreCase))
```

Comment:
what is that??

Reply:
if there was no intersection between the change vectors (we left with the first one) we should return null here

Reply:
so the fix should be in `MergeChangeVectorDown` and not here